monai/utils/deprecated.py

```diff
@@ -34,40 +34,12 @@ def warn_deprecated(obj, msg):
     Issue the warning message `msg` only once per process for the given object `obj`. When this function is called
     and `obj` is not in `warned_set`, it is added and the warning issued, if it's already present nothing happens.
     """
+    global warned_set
```

Comment:
I try to add a `global` here but not sure what would be the intended behaviour? @ericspod